test/repository_test.rb

```diff
@@ -117,6 +118,34 @@
         end
       end
 
+      describe '.created_at' do
+
+        before do
+
+          @user = UserRepository.create(user1)
+          @timeago = DateTime.new(2015,4,1)
+        
+        end
+
+        let(:article4) { Article.new(user_id: @user.id, title: 'Love Relationships', comments_count: '4') }
+        let(:article5) { Article.new(user_id: @user.id, title: 'Love Relationships', comments_count: '4', created_at: @timeago) }
+
+        it 'persisted created_at which un assign time' do
+          result = ArticleRepository.create(article4)
+          timenow = result.created_at
+          result.instance_variable_get(:@created_at).must_equal timenow
```

Comment:
Then you should be able to call the attribute getter instead of doing the instance variable getter like above